### PR TITLE
Add default keymap

### LINKED ISSUE #597: Keymap is gone
Looks like the very convenient keyboard shortcuts to jump from one "marked as error or warning line" to another are gone, now.

Would it be possible to have them available again, please?

### PR DESCRIPTION
**Pull request by steelbrain**
Fixes #597
SideNote: What does `ctrl-k` mean?

### CODE REVIEW

**Comment by k2b6s9j** [comment]
`ctrl-k` means you press both your `Control` and `K` keys on your keyboard in one stroke. /s

File: keymaps/linter.cson [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  'ctrl-cmd-l': 'linter:lint'
+  'ctrl-cmd-e': 'linter:next-error'
+
+'.platform-linux atom-text-editor:not(.mini)':
+  'ctrl-k ctrl-l': 'linter:lint'
+  'ctrl-k n': 'linter:next-error'
+
+'.platform-windows atom-text-editor:not(.mini)':
+  'ctrl-k ctrl-l': 'linter:lint'
+  'ctrl-k n': 'linter:next-error'
```
Comment (iam4x) [inline keymaps/linter.cson]: No end line?

**Comment by iam4x** [comment]
ctrl /= cmd on mac, how to have the compatibility?

File: keymaps/linter.cson (hunk 1 above)
Comment (steelbrain) [inline keymaps/linter.cson]: Added. I am not used to EOLs :smiley:

**Comment by steelbrain** [comment]
I just copy/pasted it from the old one, I think we should modify the keys. Matching the old ones are not really necessary.

**Comment by lierdakil** [comment]
Um. Looks a little bit weird? Why is it (essentially) ctrl-alt-l and ctrl-alt-e on OSX and a sequence of keystrokes for everything else?

**Comment by steelbrain** [comment]
`ctrl-n` is already used for opening a new file, do we really need a key binding for the next error, can't we just jump?

**Comment by lierdakil** [comment]
ctrl-l is already used to select current/next line by core.

**Comment by iam4x** [comment]
@steelbrain Look on `README.md` with #598 I added.

```
'atom-workspace':
  'linter:next-error': 'ctrl-alt-e'
  'linter:next-warning': 'ctrl-alt-w'
  'linter:toggle': 'ctrl-alt-t'
  'linter:set-bubble-transparent': 'alt'
```

**Comment by lierdakil** [comment]
And I'm not convinced `linter:lint` should have a default shortcut at all, that's not a very common use-case, I don't think.

**Comment by steelbrain** [comment]
So here's my solution, we only add the peek default keymap and leave the others to those who want to use them, 'cause not everyone uses those features.

**Comment by lierdakil** [comment]
I suppose `next-message` something like that could be useful as well -- that's something many would use, I think. Thoughts?

**Comment by steelbrain** [comment]
@lierdakil suggested key please?

**Comment by lierdakil** [comment]
IDK. `alt-.`/`cmd-.`? Not sure if `cmd-.` is taken on OSX or not though.
`.` because it's "lowercase" `>`

**Comment by lierdakil** [comment]
Okay, scratch that about `cmd-.`, it could be `ctrl-.` on OSX.

**Comment by lierdakil** [comment]
I guess it would be better to actually ask OSX user if such shortcut makes sense or not though.

**Comment by k2b6s9j** [comment]
OS X user here, what doesn't make sense?

**Comment by lierdakil** [comment]
@k2b6s9j, `ctrl-.` as a shortcut for `linter:next-error`.
(`.` because on PC keyboard, shifted it's `>`)

**Comment by k2b6s9j** [comment]
For me `alt-shift-,` and `alt-shift-.` make sense for previous and next error.

**Comment by k2b6s9j** [comment]
`.` on my MacBook Pro shifted is still `>`

**Comment by lierdakil** [comment]
Well, that makes sense on PC as well. @steelbrain ^

**Comment by k2b6s9j** [comment]
It evens works on Chromebooks!

**Comment by steelbrain** [comment]
Sounds good to me

**Comment by k2b6s9j** [comment]
:shipit:?

**Pull request merged**